_functions_matrix.sh

```diff
@@ -62,6 +62,8 @@ do_start_matrix() {
   fi
   mkdir -p ${DEPLOYMENT_DIR}/matrix
   evaluate_file_content ${ETC_DIR}/matrix/homeserver.yaml.template ${DEPLOYMENT_DIR}/matrix/homeserver.yaml
+  evaluate_file_content ${ETC_DIR}/matrix/initialize.sh ${DEPLOYMENT_DIR}/matrix/initialize.sh
```

Comment:
```suggestion
  evaluate_file_content ${ETC_DIR}/matrix/initialize.sh.template ${DEPLOYMENT_DIR}/matrix/initialize.sh
```
Rename also the file initialize.sh to initialize.sh.template